src/libData/AccountData/AccountStoreSC.tpp

```diff
@@ -350,119 +517,125 @@ bool AccountStoreSC<MAP>::UpdateAccounts(const uint64_t& blockNum,
         if (ret) {
           std::string runnerPrint;
 
-          // invoke scilla runner
-          InvokeInterpreter(RUNNER_CREATE, runnerPrint, scilla_version,
-                            is_library, gasRemained, amount, ret, receipt);
-
-          // parse runner output
-          try {
```

Comment:
Dont see this code block in the new changes, any reason why?

Reply:
Code compared with 8.2 and master and original code restored, the change was in error and served no useful purpose.